ql/src/test/queries/clientpositive/querycompilesummaryhook.q

```diff
@@ -0,0 +1,11 @@
+set hive.query.lifetime.hooks=org.apache.hadoop.hive.ql.QueryCompilationSummaryHook;
+
+create table tbl_n1 (n bigint, t string); 
+
+insert into tbl_n1 values (1, 'one'); 
+insert into tbl_n1 values(2, 'two');
+
+explain
+select a.n, a.t, isnull(b.n), isnull(b.t) from (select * from tbl_n1 where n = 1) a  left outer join  (select * from tbl_n1 where 1 = 2) b on a.n = b.n;
```

Comment:
Do we need a complex query for showcasing the summary. If not then better simplify it.